portfolio.html

```diff
@@ -52,28 +58,42 @@ <h2 class="portfolio-title">
 				<div class="dotted-border">
```

Comment:
I would suggest being more specific with the div names here. Perhaps call the first div `portfolio-parent` and instead of calling the child dics `dotted-border` you could try `portfolio-card`